chess2.h

```diff
@@ -0,0 +1,49 @@
+#ifndef CHESS2_H_INCLUDED
+#define CHESS2_H_INCLUDED
+
+#include<vector>
+#include<string>
+
+
+typedef std::vector<std::string> buff;
+typedef std::string str;
+
+class Chess2
+{
+public:
+    buff pseudomoves;
+    buff legal_moves;
+    short int board[8][8] = // This array represents the chess board
+    {
+      {-5,0,0,-6,-10,-2,-3,-5},
```

Comment:
Define constants numbers for the items like pawn, knight, king etc and use that for easy to look after.